docs/application/dotnet/guides/applications/uiapplication/component-based-app.md

```diff
@@ -308,7 +308,7 @@ Following are the main application attributes:
 
 ## Prerequisites
 
-To use the functions and data types of the component based application API (in [mobile](/application/dotnet/api/mobile/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) and [wearable](/application/dotnet/api/wearable/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) applications), include the `Tizen.Applications.ComponentBased.Common` module in your application:
+To use the functions and data types of the [Tizen.Applications.ComponentBased.Common](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.Common.html) namespace, include it in your application:

```

Comment:
Hello @mijin85cho, hope you're doing well.

As per our discussion on https://github.com/Samsung/tizen-docs/pull/1734#discussion_r1045541882, I have made the necessary change.

Please check the change made, thank you.